src/azure-cli/HISTORY.rst

```diff
@@ -3,13 +3,16 @@
 Release History
 ===============
 
+**SQL**
+
+* Added "--read-scale" and "--read-replicas", parameters to support read scale management with SQL Database CRUD operations."
```

Comment:
Why do you end with "? typo?

Reply:
Yes, a typo.  I am also updating this comment to specify sql db create and update commands.

Reply:
Fixed